assets/app/view/spreadsheet.rb

```diff
@@ -168,5 +186,17 @@ def render_player_certs
         *@game.players.map { |p| h(:td, p.num_certs) },
       ])
     end
+
+    def corporation_operated?(corporation, current_round)
+      current_round_s = current_round.join('.')
```

Comment:
corperation.operating_history.any? { |history| history[0].join('.') == current_round }

Reply:
if you keep it as an array, do you even need to join the history here? can’t you just compare arrays?

Reply:
My ruby is a little weak but I would like to eject from the function and return true if found. Not sure if the lamda will help me here :(

Reply:
Ok I tried it as you suggested but it does not return true for the case when found - this is what I tried
```
  def corporation_operated?(corporation, current_round)
      current_round_s = current_round.join('.')
      corporation.operating_history.any? { |history| history[0].join('.') == current_round }
    end
```
also
```
  def corporation_operated?(corporation, current_round)
      current_round_s = current_round.join('.')
      corporation.operating_history.any? { |history| return true if history[0].join('.') == current_round }
    end
```

Reply:
you can replace the whole thing with the lambda

corperation.operating_history.any? { |history| history[0] == current_round }

this will return true if it has a match else false